mainapp/models.py

```diff
@@ -68,6 +68,7 @@
     ('L', 'Low')]
 
 
+phone_number_regex = '^((\+91|0)[\- ]{0,1})?[456789]\d{9}$'
```

Comment:
This change might be confusing, Because various regex are used across models (Volunteer vs NGO). Better to define validator and field inside model, Similar to NGO model.

https://github.com/IEEEKeralaSection/rescuekerala/blob/master/mainapp/models.py#L174-L175

Reply:
Done @salmanulfarzy  👍

Reply:
Done @salmanulfarzy 👍